sql/parser/sql.y

```diff
@@ -1536,22 +1536,15 @@ transaction_mode_list_or_empty:
 // | /* EMPTY */ {}
 
 create_database_stmt:
-  CREATE DATABASE name createdb_opt_with createdb_opt_list
+  CREATE DATABASE name
   {
-    // TODO(pmattis): Handle options.
     $$ = &CreateDatabase{Name: Name($3)}
   }
-| CREATE DATABASE IF NOT EXISTS name createdb_opt_with createdb_opt_list
+| CREATE DATABASE IF NOT EXISTS name
   {
-    // TODO(pmattis): Handle options.
     $$ = &CreateDatabase{IfNotExists: true, Name: Name($6)}
   }
 
-createdb_opt_with:
-  WITH {}
-| WITH_LA {}
-| /* EMPTY */ {}
-
 createdb_opt_list:
```

Comment:
I think you can remove `createdb_opt_list`, `createdb_opt_items` and `createdb_opt_item` as well.